developer-docs-site/docs/concepts/staking.md

```diff
@@ -39,13 +35,22 @@ Below is a summary flow diagram of how staking on the Aptos blockchain works. Th
 
 ### How a custodian can stake on Aptos
 
-The Aptos staking module defines a capability that represents ownership. See [https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85](https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85).
+The Aptos staking module defines a capability that represents ownership. 
+
+:::tip Ownership
+See [https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85](https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85).
```

Comment:
We should in general fix these to use the branch `main` since the commit will eventually be old